### PR TITLE
LOG-4184: fix config forwarding to default Loki using vector

### PR DESCRIPTION
**Pull request by vparfonov**
### Description

/cc 
/assign 

/cherry-pick 

### Links

- Depending on PR(s):
- Bugzilla:
- Github issue:
- JIRA:
- Enhancement proposal:

### CODE REVIEW

File: internal/generator/vector/output/loki/loki_conf_test.go [hunk 1]
```diff
@@ -659,105 +859,107 @@ var _ = Describe("Generate vector config for in cluster loki", func() {
 	var f = func(clspec logging.CollectionSpec, secrets map[string]*corev1.Secret, clfspec logging.ClusterLogForwarderSpec, op generator.Options) []generator.Element {
 		return Conf(clfspec.Outputs[0], inputPipeline, secrets[constants.LogCollectorToken], generator.NoOptions)
 	}
-	DescribeTable("for Loki output", helpers.TestGenerateConfWith(f)) //		Entry("with default logcollector bearer token", helpers.ConfGenerateTest{
-	//			CLFSpec: logging.ClusterLogForwarderSpec{
```
Comment (jcantrill) [inline internal/generator/vector/output/loki/loki_conf_test.go:663]: Remove me if not required

File: internal/generator/vector/output/loki/loki_conf_test.go [hunk 2]
```diff
@@ -659,105 +859,107 @@ var _ = Describe("Generate vector config for in cluster loki", func() {
...
-	//`,
-	//		}),
+	DescribeTable("for Loki output", helpers.TestGenerateConfWith(f))
+	Entry("with default logcollector bearer token", helpers.ConfGenerateTest{
+		CLFSpec: logging.ClusterLogForwarderSpec{
+			Outputs: []logging.OutputSpec{
+				{
+					Type: logging.OutputTypeLoki,
+					Name: "loki-receiver",
+					URL:  "http://lokistack-dev-gateway-http.openshift-logging.svc:8080/api/logs/v1/application",
```
Comment (jcantrill) [inline internal/generator/vector/output/loki/loki_conf_test.go:869]: Won't internal always be https?  Can we use http with certs?

File: internal/generator/vector/output/loki/loki_conf_test.go (hunk 1 above)
Comment (vparfonov) [reply internal/generator/vector/output/loki/loki_conf_test.go:663]: this code was commented by mistake in previous commit

**Pull request merged**